Packs/FeedUnit42v2/Integrations/FeedUnit42v2/FeedUnit42v2.py

```diff
@@ -123,11 +123,13 @@ def parse_indicators(indicator_objects: list, feed_tags: Optional[list] = None,
     indicators = []
     if indicator_objects:
         for indicator_object in indicator_objects:
+            raw_name = indicator_object.get('name', '')
             pattern = indicator_object.get('pattern') or ''
+            
             for key in UNIT42_TYPES_TO_DEMISTO_TYPES.keys():
                 if pattern.startswith(f'[{key}'):  # retrieve only Demisto indicator types
                     indicator_obj = {
-                        "value": indicator_object.get('name', ''),
+                        "value": raw_name,
                         "type": UNIT42_TYPES_TO_DEMISTO_TYPES.get(key),
                         "score": ThreatIntel.ObjectsScore.MALWARE,  # default verdict of fetched indicators is malicious
```

Comment:
```suggestion
                        "score": DEFAULT_INDICATOR_SCORE,  # default verdict of fetched indicators is malicious
```

Please add a constant value DEFAULT_INDICATOR_SCORE at the start of the file and use it

Reply:
done